app/models/manageiq/providers/openstack/infra_manager/host.rb

```diff
@@ -412,11 +412,119 @@ def provide(taskid = nil)
       begin
         MiqEvent.raise_evm_job_event(self, :type => "provide", :suffix => "complete")
       rescue => err
-        $log.warn("Error raising complete provide event for #{log_target}: #{err.message}")
+        _log.warn("Error raising complete provide event for #{log_target}: #{err.message}")
       end
     end
 
     task.update_status("Finished", task_status, "Provide Complete with #{workflow_state}") if task
-    $log.info("Provide #{log_target}...Complete - Timings: #{t.inspect}")
+    _log.info("Provide #{log_target}...Complete - Timings: #{t.inspect}")
+  end
+
+  def supports_start?
```

Comment:
@rwsu please use `validate_start` in this PR, it's defined in the base `host.rb` class, so you can override that method in here.

For the `supports_` style we have to convert a feature in one go, in order to make it consistent.